### ISSUE TITLE
Reproductibility issue: manual seed

### ISSUE THREAD

**Issue by KrishnaPriyaIITR** [amaiya/ktrain#147]
Is there a way to add manual seed value while training a model so that we get same score on evaluation set every time with same parameters and seed. Right now same parameter yields different output every time. Please help.

**Comment by amaiya** [comment]
You try to do something [like this](https://stackoverflow.com/questions/48631576/reproducible-results-using-keras-with-tensorflow-backend), but never tried it.  Some posts indicate that you can only do this by disabling GPU, so not sure it's worth it.  If you're trying to compare results of different models, the alternative is to take the average across runs. See also [this](https://www.kaggle.com/c/mercari-price-suggestion-challenge/discussion/45502).
